src/ui/components/Library/Team/View/Tests/Overview/TestErrorList.tsx

```diff
@@ -47,6 +48,8 @@ export function TestErrorList({
   );
 
   const uniqueErrorCount = Object.entries(errorFrequency).length;
+  // We don't bother displaying executions without replays
+  const filteredExecutions = executions.filter(e => e.recording[0] !== null);
```

Comment:
There is an incoming backend fix that will make it so that we don't have to filter by null anymore